### ISSUE TITLE
Cypress integration tests include and pass accessibility checks

### ISSUE THREAD

**Issue by Sruthin86** [MESH-Research/CCR#180]
### Problem/Why

Cypress integration tests does not include accessibility checks for several pages and these pages are not 100% accessible 

### Goal/What

All pages should include Cypress-axe accessibility checks and should pass the checks.

### Solution/How

Add the accessibility checks in the integration tests and fix the issues that are reported out.
```
// Basic usage
it('Has no detectable a11y violations', () => {
  // Test the page at initial load
  cy.checkA11y()
})
```
https://www.npmjs.com/package/cypress-axe
### To Do / Acceptance Criteria

- [x] All the integration tests have accessibility checks on `account.spec.js`, `header.spec.js`, `login.spec.js` .
- [x] All the pages pass the Cypress-axe integration checks. 

### Review Checklist

- [x] All the integration tests have accessibility checks on `account.spec.js`, `header.spec.js`, `login.spec.js`.
- [x] All the pages pass the Cypress-axe integration checks.

### Accessibility Checklist

- [ ]

**Comment by gmeben** [comment]
Today I took care of all outstanding errors as reported by the Chrome axe DevTools extension for all implemented pages as well as the dropdown navigation.
